### PR TITLE
Refactor vm spinup

### PR DESCRIPTION
**Pull request by dougbtv**
Terse changes to simply remove roles and change spinup scripts.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -13,3 +13,4 @@ group_vars/
 roles/install-go
 roles/install-docker
 roles/auto-kube-dev
+roles/redhat-nfvpe.vm-spinup
```
Comment (leifmadsen) [inline .gitignore:16]: oh snap you remembered :)

File: requirements.yml [hunk 2]
```diff
@@ -8,3 +8,5 @@
 - src: https://github.com/redhat-nfvpe/auto-kube-dev
   name: auto-kube-dev
   version: master
+- name: redhat-nfvpe.vm-spinup
+  version: master
```
Comment (leifmadsen) [inline requirements.yml:12]: we could also tag a new version and pin it here? I'm ok with master for now though.

**Review by leifmadsen — APPROVED** [review]
I think this looks spot on. We can merge this, and I'll test/iterate on any possible issues.

File: requirements.yml (hunk 2 above)
Comment (dougbtv) [reply requirements.yml:12]: sounds good, maybe worth pinning in the future if one leap frogs

**Pull request merged**